src/DynamoPackages/Package.cs

```diff
@@ -599,6 +600,8 @@ internal void UninstallCore(CustomNodeManager customNodeManager, PackageLoader p
                 if (BuiltInPackage)
                 {
                     LoadState.SetAsUnloaded();
+                    Analytics.TrackEvent(Actions.BuiltInPackageConflict, Categories.PackageManagerOperations, $"{Name } {versionName} set unloaded");
```

Comment:
@mjkkirschner what indicates that this code path is a package conflict scenario?